leasing/report/invoice/laske_invoice_count_report.py

```diff
@@ -85,7 +90,24 @@ def get_data(self, input_data):
                 for due_date in due_dates:
                     data[due_date - relativedelta(months=1)] += 1
 
-        return [
-            {"send_date": i[0], "invoice_count": i[1], "is_estimate": i[0] >= today}
-            for i in data.items()
-        ]
+            # Creating (credit) invoices with due dates in the past seems to be quite common.
+            # We can't just choose all with sent_to_sap_at__isnull=True, because that's the default value...
+            invoices = Invoice.objects.filter(
+                due_date__lte=due_dates_end,
+                due_date__gte=due_dates_start,
+                sent_to_sap_at__isnull=True,
+            )
+            for invoice in invoices:
+                data[invoice.due_date - relativedelta(months=1)] += 1
+
+        send_dates = []
+        for i in data.items():
```

Comment:
Now that this is not a list comprehension, you could make a bit clearer. e.g. (not tested)

```python
for send_date, invoice_count in data.items():
    if invoice_count == 0:
        continue

    send_dates.append(
        {
            "send_date": send_date,
            "invoice_count": invoice_count,
            "is_estimate": send_date >= today,
        }
    )
```